Js2

mathiasg12 opened the pull request.

mathiasg12 committed:
v1

startared CA, made register and login to the api

mathiasg12 committed:
v2

started on feed page and made get request to fetch posts

mathiasg12 committed:
v3

mathiasg12 committed:
v4

added searach and filter function

mathiasg12 committed:
v4

added possibility for users to add a post

mathiasg12 committed:
v5

margin fix

mathiasg12 committed:
v6

added specific page

mathiasg12 committed:
v7

added profile page and delete, edit button (they are not working yet)

mathiasg12 committed:
v8

delete and update post functionality now works

mathiasg12 committed:
v9

added form validation to landing page

mathiasg12 committed:
v10

added filter, view more post and some styling

mathiasg12 committed:
v11

improved landing page

mathiasg12 committed:
v11.5

design inprovments, added logout button

mathiasg12 committed:
v12

added search button

mathiasg12 committed:
v13

improved posts, added loading bar

mathiasg12 committed:
v14

added global search so you can search the whole api

mathiasg12 committed:
v15

filter and sort now works with search results aswell as more button

mathiasg12 committed:
v15.1

deleted console logs

mathiasg12 committed:
v15.2

added README.md

Fermain reviewed (commented):
While there are areas for improvement, the basic features needed to pass have been delivered. Things to think about for improvement:

- Error handling, what happens to the user when an error occurs? Do they see a nice message?
- Size of JS files, are there ways to split your code up into a smaller main file to make it easier to read?
- Please double check the readme file for correct markdown syntax

mathiasg12 committed:
15.3

fixed readme syntax

mathiasg12 committed:
v15.2

improved README.md

mathiasg12 committed:
15.3

added error message to api functions and fixed responsive issue on feedpage

mathiasg12 committed:
v15.4

added new file

mathiasg12 committed:
v15.5

splitting  js files into smaller files

mathiasg12 committed:
v15.6

made more js files to keep them at a more read able size, also added js docs to all functions

mathiasg12 committed:
v15.7

splitting js in smaller files, fixing some typos

mathiasg12 committed:
Merge pull request #4 from mathiasg12/js2-bugfixing

Js2 bugfixing
